onnxruntime/python/tools/transformers/models/stable_diffusion/ort_utils.py

```diff
@@ -7,16 +7,28 @@
 import logging
 import os
 import shutil
+import sys
 from typing import Union
 
 import torch
 
 import onnxruntime as ort
-from onnxruntime.transformers.io_binding_helper import CudaSession
 
 logger = logging.getLogger(__name__)
 
 
+def add_transformers_dir_to_path():
+    sys.path.append(os.path.dirname(__file__))
+
+    transformers_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", ".."))
```

Comment:
Can the scripts be called from source as `python -m models.stable_diffusion.<script-name>` and then remove the path append? The path appends were removed in [this PR](https://github.com/microsoft/onnxruntime/pull/17179).

Reply:
Some users might prefer to run the script directly in current directory for convenience  (instead of go to parent directory), so I added the support (drawback is code looks less clean).

Reply:
If users want to run the scripts directly, can we move the path append function elsewhere to benefit the other model scripts?

Reply:
Yes. Let me add another PR to address that (also enable it in other models)